book/02-git-basics/sections/viewing-history.asc

```diff
@@ -155,13 +155,13 @@ ca82a6d - Scott Chacon, 6 years ago : Change version number
 a11bef0 - Scott Chacon, 6 years ago : Initial commit
 ----
 
-<<pretty_format>> listet einige der nützlichsten Optionen auf, die `format` bietet.
+<<pretty_format>> listet einige der nützlichsten Spezifkationen auf, die `format` bietet.
```

Comment:
```suggestion
<<pretty_format>> listet einige der nützlichsten Spezifkatoren auf, die `format` bietet.
```

Es gäbe noch `Spezifikationssymbol`, das ist mir aber eindeutig zu gestelzt 😄 
Genauso würde ich vorschlagen, die restlichen Vorkommnisse von `Spezifikation` durch `Spezifikator` zu ersetzen.

Reply:
Da ist doch ein Typo drin "Spezifkatoren"  --> "Spezifikatoren" ??

Reply:
Hast recht!